### ISSUE TITLE
PushPlugin does not compile for pgb cli-6.0.0 (iOS 4.0.1).

### ISSUE THREAD

**Issue by kleeb** [phonegap-build/PushPlugin#674]
Latest Phonegap Build release cli-6.0.0 comes with cordova-ios 4.0.1.
On this version I am getting:

`Build error : CompileC /Release-iphoneos/SlayerStG.build/Objects-normal/armv7/PushPlugin.o /Plugins/com.phonegap.plugins.PushPlugin/PushPlugin.m normal armv7 objective-c com.apple.compilers.llvm.clang.1_0.compiler`

on cli-5.4.1 everything compiles ok.

any clues?

**Comment by kaplanmaxe** [comment]
+1. Any update?

**Comment by macdonst** [comment]
@kleeb @kaplanmaxe why do people keep using this deprecated plugin? It is no longer under active maintenance. We have a new plugin over at https://github.com/phonegap/phonegap-plugin-push that I'm working very hard at to stay current.

Just wondering what the impediments are for people to switch over.

**Comment by kaplanmaxe** [comment]
@macdonst thanks for your continued hard work on this plugin.

The attached link points to this repo.

http://ngcordova.com/docs/plugins/pushNotifications/

**Comment by macdonst** [comment]
@kaplanmaxe well, I know the new Ionic Push stuff relies on the new push plugin over at:

https://github.com/phonegap/phonegap-plugin-push

I'm even working with @ericb from Ionic on it to support Ionic users better. You really should switch.

**Comment by kaplanmaxe** [comment]
No worries, I already switched. Was just pointing out how I got to the repo in the first place.

**Comment by kleeb** [comment]
@macdonst  We are currently under the process of changing the whole stack, so in the meantime until its finished we are just maintaining what has been done so it keeps working.

**Comment by danicholls** [comment]
Yeah, there's no indication on PGB that this is deprecated, or anything but (apparently) a semi-official PGB plugin.
